server/dash/dashboards/neighborhood_recent.py

```diff
@@ -149,12 +159,19 @@ def populate_options():
     Input("council_list", "value")
 )
 def update_table(selected_council):
-    """Dash Callback Function to update the LA 311 request data table at the bottom of the dashboard.
+    """Filters the LA 311 request data table based on selected_council.
 
     This function takes the selected neighborhood council (nc) value from the "council_list" dropdown and 
     outputs a list of requests associated with that nc as a data table in dictionary form 
     with id "council_table" in the layout. 
 
+    Args:
+        selected_council:
+            A string argument automatically detected by Dash callback function when "council_list" element is selected in the layout.
+
+    Returns:
+        A list of dictionary mapping column names to values. For example: [{'srnumber':1234567}, {'createdDate':'2022-07-11'}, {'closeDate':'2022-07-14'}...]
```

Comment:
Is this true? I would expect the output to be like:
`[{'srnumber':1234567, 'createdDate':'2022-07-11', 'closeDate':'2022-07-14'...}, {...}, ... ]`

I.e., a dictionary for a single record (multiple k:v's in a dict) instead of a dict per value.

Reply:
My bad - I read the documentation wrong. Your example is correct!